locations/spiders/denmans_gb.py

```diff
@@ -0,0 +1,50 @@
+from scrapy import Spider
+
+from locations.dict_parser import DictParser
+from locations.hours import OpeningHours
+
+
+class DenmansGBSpider(Spider):
+    name = "denmans_gb"
+    item_attributes = {"brand": "Denmans", "brand_wikidata": "Q116508855"}
+    # Seems to return all stores regardless of lat long as long as it's in the UK?
+    start_urls = ["https://www.denmans.co.uk/den/store-finder/findNearbyStores?latitude=51&longitude=-0"]
+
+    def parse(self, response):
+        for store in response.json()["results"]:
+            store["address"]["street_address"] = ", ".join(
+                filter(
+                    None,
+                    [
+                        store["address"].pop("line1"),
+                        store["address"].pop("line2"),
+                        store["address"].pop("line3"),
+                    ],
+                )
+            )
+            item = DictParser.parse(store["address"])
+            item["phone"] = store["address"]["phone"]
+            item["contact:email"] = store["address"]["email"]
```

Comment:
It's just `email`, you can see the full list [here](https://github.com/alltheplaces/alltheplaces/blob/master/locations/items.py).

But in this case, `DictParser` should get the email and phone, so you can delete these lines.
```suggestion
```

Reply:
Thanks, I've just opened https://github.com/alltheplaces/alltheplaces/pull/4702 to fix the docs!